Doppler.ContactPolicies.Api.Test/Authorization.cs

```diff
@@ -73,7 +73,7 @@ public async Task
 
             var contactPoliciesMock = new Mock<IContactPoliciesService>();
             contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(accountName)).ReturnsAsync(It.IsAny<int>());
-            contactPoliciesMock.Setup(x => x.GetContactPoliciesSettingsAsync(accountName)).ReturnsAsync(contactPoliciesSettings);
+            contactPoliciesMock.Setup(x => x.GetContactPoliciesSettingsAsync(It.IsAny<int>())).ReturnsAsync(contactPoliciesSettings);
```

Comment:
What do you think about to create a fake `idUser` ?
`var userId = fixture.Create<int>();`

Then we can use that in the setup:
 ```
contactPoliciesMock.Setup(x => x.GetIdUserByAccountName(accountName)).ReturnsAsync(userId);
contactPoliciesMock.Setup(x => x.GetContactPoliciesSettingsAsync(userId)).ReturnsAsync(contactPoliciesSettings);
```
```suggestion
            contactPoliciesMock.Setup(x => x.GetContactPoliciesSettingsAsync(idUser)).ReturnsAsync(contactPoliciesSettings);
```

Reply:
Good idea @ms-darianbenito it makes the test more robust because also tests the input value.

Using _AutoFixture_ would be nice, but if @jkortega9008 does not want to add the dependency, a hardcoded value could also work.

Reply:
i think is the same think we have right now, but if you prefer the variable i can put it in there.

> Good idea @ms-darianbenito it makes the test more robust because also tests the input value.
to do this we have to something else just to be aware is a positive number as input IdUser

Reply:
my bad, AutoFixture always return natural numbers

Reply:
Perdón, salto a español.

Pensé que no dependíamos de autofixture y que tal vez no querías agregar una nueva dependencia y por eso proponía la constante. No lo veía como algo muy relevante.

El comentario de Darian, en cambio, creo que si es muy importante, porque permite verificar que el parámetro con el que se invoca `GetContactPoliciesSettingsAsync` sea el correcto.

Creo que ahora está bien.